core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksBlockMetaStoreMetaOnly.java

```diff
@@ -57,7 +59,10 @@ public List<BlockLocation> getLocations(long blockid) {
 
   @Override
   public void addLocation(long blockId, BlockLocation location) {
-    mBlockLocations.addInnerValue(blockId, location.getWorkerId(), location);
+    // NOTICE(maobaolong): mLocationCacheMap can be increase to WOKRER_COUNT X TIER_COUNT
```

Comment:
consider when to remove from this map

Reply:
We can remove in LostWorkerDetectionExecutor heartbeat thread, when a worker is removed from `mWorkers` into `mLostWorkers`, we remove all worker location objects after `processLostWorker()`